### PR TITLE
docs(added postman-sdk documentation): added postman-sdk documentation

### PR DESCRIPTION
**Pull request by cedrabner**
# Pull request checklist

Use the following template to make sure your PR fits the ConsenSys documentation standard.

## Before creating the PR

Make sure that:

- [x] You've read the [contribution guidelines](https://docs-template.consensys.net/getting-started/style-guide).
- [x ] You've previewed your changes locally.

## Describe the change

Updated the links that went to blockscout and changed them to the goerli.lineascan.build

## Issue fixed

No current issue exists

## Impacted parts

For content changes:

- [ x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] Build and QA tools
- [ ] Node dependencies and JavaScript
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've fixed any issues raised by the tests.
- [ ] You've previewed your changes on Vercel below.

### CODE REVIEW

File: docs/developers/quickstart/postman-sdk.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
+---
```
Comment (emilyJLin95) [inline docs/developers/quickstart/postman-sdk.md:1]: I don't think this should go under quickstart

quickstart contains info for devs deploying contracts, not using the SDK
Would be okay with taking away quickstart and changing to deploy your contracts or some other folder structure if you think it's better

File: docs/developers/quickstart/postman-sdk.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+## Installation
+
+To install the package you need to run:
+
+```bash
+npm install @consensys/linea-sdk
+```
+
+## Implementation
```
Comment (emilyJLin95) [inline docs/developers/quickstart/postman-sdk.md:20]: I'm wondering if we should create a sample repo that would have all this code?
https://github.com/ConsenSys/linea-tutorials
This has existed for a while, but I haven't been good about updating it

File: docs/developers/quickstart/postman-sdk.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+```typescript
+import { PostmanServiceClient } from "@consensys/linea-sdk";
+import { format, transports } from "winston";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+async function main() {
+  const client = new PostmanServiceClient({
+    l1Config: {
```
Comment (emilyJLin95) [inline docs/developers/quickstart/postman-sdk.md:33]: Do we have documentation on what each of these parameters are

**Review by emilyJLin95 — CHANGES_REQUESTED** [review]

File: docs/developers/quickstart/postman-sdk.md (hunk 1 above)
Comment (mapachurro) [reply docs/developers/quickstart/postman-sdk.md:1]: I'm OK with this plan:
-Rename 'quickstart' to 'Deploy a contract'
-Create an 'SDK' section

Alternatively, the 'quickstart' section stays, and within it, we create a section for 'Deploy a contract' and 'SDK'

File: docs/developers/quickstart/postman-sdk.md (hunk 2 above)
Comment (mapachurro) [reply docs/developers/quickstart/postman-sdk.md:20]: If the team is good with publicizing this, I think it's a great idea, esp for that growing group of devs that's working on zkEVM architecture/high level functionality

File: docs/developers/quickstart/postman-sdk.md (hunk 3 above)
Comment (cedrabner) [reply docs/developers/quickstart/postman-sdk.md:33]: the .env file explains what all the parameters are if that's what you are asking?

**Review by mapachurro — APPROVED** [review]
Hell yeah! I'm excited for this.

**Review by mapachurro — APPROVED** [review]

**Review by mapachurro — APPROVED** [review]
From what I can tell, Chris had addressed Emily's concerns here. We should continue thinking about how best to structure the docs overall.

**Review by emilyJLin95 — APPROVED** [review]
LGTM

**Pull request merged**